data/systemd/Makefile

```
@@ -34,7 +39,8 @@ check: snapd.run-from-snap snapd.core-fixup.sh
 install: $(SYSTEMD_UNITS)
 	# NOTE: old (e.g. 14.04) GNU coreutils doesn't -D with -t
 	install -d -m 0755 $(DESTDIR)/$(SYSTEMDSYSTEMUNITDIR)
-	install -m 0644 -t $(DESTDIR)/$(SYSTEMDSYSTEMUNITDIR) $^
+	install -m 0644 -t $(DESTDIR)/$(SYSTEMDSYSTEMUNITDIR) $(filter-out snap.mount, $^)
+	install -m 0644 snap.mount $(DESTDIR)/$(SYSTEMDSYSTEMUNITDIR)/$(SNAP_MOUNT_UNIT).mount
```

Can you add a comment like this:
```suggestion
        # The mount unit name must reflect the actual path of snap mount directory.
	install -m 0644 snap.mount $(DESTDIR)/$(SYSTEMDSYSTEMUNITDIR)/$(SNAP_MOUNT_UNIT).mount
```